tests/test_core.py

```diff
@@ -879,6 +879,7 @@ def test_scaper_instantiate():
     # we get back is as expected.
     sc = scaper.Scaper(10.0, fg_path=FG_PATH, bg_path=BG_PATH)
     sc.ref_db = -50
+    sc.sr = 22050
```

Comment:
What's the purpose of setting sr explicitly here?

Reply:
I was testing that it could be saved and reloaded as a value other than the default.

Reply:
Yes, that makes sense, but right now it's passing even though the regression jam files doesn't contain an sr field. Let's keep this line and add `sr: 22050` to the regression jam file. One we fix  `_compare_scaper_jams` adding `np.union1d` as per the above comment this check should work (perhaps first fix the function, see that this test fails, and only then update the regression jams file to confirm that the test now passes).